src/engine/main/Engine.C

```diff
@@ -807,15 +803,7 @@ Engine::InitializeCompute()
 #endif
     if (avtCallback::GetSoftwareRendering())
     {
-        // Install factory for  VisIt's OffScreen Render Window overrides
-#if LIB_VERSION_LE(VTK,9,2,6)
-#ifdef HAVE_OSMESA
-        debug1 << mName << "Offscreen rendering will use offscreen factory." << endl;
-        vtkOffScreenRenderingFactory::ForceOffScreen();
-#else
-        debug1 << mName << "Offscreen rendering will use GL." << endl;
-#endif
-#endif
+        // any message?
```

Comment:
any message?

Reply:
Thanks, that was a note to remind me to figure out if there should be a message here. I cannot think of one because VTK now determines the backend for offscreen rendering, which I don't think I can query at this point.

Reply:
Now I'm wondering if the `SoftwareRendering` flag is still needed.  Will investigate.